db/models.py

```diff
@@ -22,6 +24,11 @@
     actors = models.ManyToManyField(to=Actor, related_name="movies")
     genres = models.ManyToManyField(to=Genre, related_name="movies")
 
+    class Meta:
+        indexes = [
+            models.Index(fields=["title"], name="movie_title_idx")
```

Comment:
There is a typo in the index name. It should be 'movie_title_idx' instead of 'muvie_title_idx'.